modules/my_preferences/README.md

```diff
@@ -0,0 +1,29 @@
+# My Preferences
+
+## Purpose
+
+This module allows a user to edit their basic profile information as well as
+customize their notification settings.
+
+## Intended Users
+
+Every user is able to modify their profile information.
+
+## Scope
+
+This module allows a user to edit their basic profile information as well as
+customize their notification settings.
+
+## Permissions
+
+A user is always able to access the My Preferences page.
+
+## Configurations
+
+Some settings related to email notifications affect this module.
```

Comment:
What settings related to email notification affect this module?

Reply:
Notification settings for different modules with upload/download. I'll modify this